problems/objects.js

```diff
@@ -168,7 +223,28 @@ const cubeObj = () => {};
  * @returns {Object} Counts of e and a. {a: 2, e: 1}
  */
 
-const countAandE = () => {};
+const countAandE = (str) => {
+  let obj = {
+    a: 0,
+    e: 0,
+  };
+
+  let sumA = 0;
+  let sumE = 0;
+
+  for (let i = 0; i < str.length; i++) {
+    if (str[i] === "a" || str[i] === "A") {
+      sumA++;
+      obj["a"] = sumA;
```

Comment:
If we can increase the value by writing obj["a"]++, do we need the variables on lines 232-233?